install

```diff
@@ -226,24 +225,24 @@ end
 
 wait_for_user if STDIN.tty? && !ENV["TRAVIS"]
 
-if File.directory? HOMEBREW_PREFIX
-  sudo "/bin/chmod", "g+rwx", *group_chmods unless group_chmods.empty?
-  sudo "/bin/chmod", "u+rwx", *user_chmods unless user_chmods.empty?
-  sudo "/usr/sbin/chown", ENV["USER"], *chowns unless chowns.empty?
-  sudo "/usr/bin/chgrp", "admin", *chgrps unless chgrps.empty?
```

Comment:
Can you explain why not? I see this flow as:
1. Ensure that HOMEBREW_PREFIX dir is created;
2. Ensure all required dirs in HOMEBREW_PREFIX are in place (create them if not)
3. Ensure all permissions are fine.

Tell me if I am missing some point.